sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -52,6 +52,8 @@ import (
 	"github.com/pulumi/pulumi/pkg/version"
 	pulumirpc "github.com/pulumi/pulumi/sdk/proto/go"
 	"google.golang.org/grpc"
+
+	"github.com/coreos/go-semver/semver"
```

Comment:
We use `blang/semver` elsewhere - is there a particular reason you picked this library?

Reply:
I don't mind either way which library we use, but we should be consistent.

Reply:
Nope  Fixing!